ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -375,18 +385,27 @@ export const getEligibleModules = (
 
 const incompleteModules = (
   modules: AccessModule[],
-  profile: Profile
+  profile: Profile,
+  pageMode: DARPageMode
 ): AccessModule[] =>
   modules.filter(
     (moduleName) =>
-      !isCompliant(getAccessModuleStatusByName(profile, moduleName))
+      !isCompliant(getAccessModuleStatusByName(profile, moduleName)) ||
+      (pageMode === DARPageMode.ANNUAL_RENEWAL &&
+        !isRenewalCompleteForModule(
+          getAccessModuleStatusByNameOrEmpty(
+            profile.accessModules.modules,
+            moduleName
+          )
+        ))
   );
 
 // exported for test
 export const getActiveModule = (
```

Comment:
we probably don't want to replicate this behavior (only one module can be the active one) for renewal ... let's think about whether this is OK as an iteration.